dhcp/install.sh

```diff
@@ -2,12 +2,33 @@
 
 source /vagrant/settings.sh
 
-#installation
+# installation
 apt --yes update
 apt --yes --install-recommends install dnsmasq
+export DEBIAN_FRONTEND=noninteractive
+apt -yq --yes install iptables-persistent
 
-#configuration
+# fetching interface name
+INTERNET=$(ifconfig | grep -m 1 RUNNING | cut -d':' -f1)
+LOCAL=$(ifconfig | grep -m 2 RUNNING | cut -d':' -f1 | tail -n1)
+
+# configuration
 echo "dhcp-range=tftp,${NETWORK}.250,${NETWORK}.254" >> /etc/dnsmasq.conf 
```

Comment:
Are you sure we need this line with `tftp`? What does the documentation of `dnsmasq` say about it?

Reply:
yes, the documentation says If dnsmasq is providing a TFTP service then only the filename is required to enable network booting. 
Some comment on stackoverflow gave an explanation to this line and concluded that it would reduce time required for network boot.

Reply:
> yes, the documentation says If dnsmasq is providing a TFTP service then only the filename is required to enable network booting.

We don't need TFTP service on the DHCP server.

> Some comment on stackoverflow gave an explanation to this line and concluded that it would reduce time required for network boot.

You cannot trust all the comments on stackoverflow. Couldn't you find anything on the documentation of dnsmasq?

Reply:
> We don't need TFTP service on the DHCP server.

Please remove this line and fix the configuration of dnsmasq.